packages/generate/package.json

```diff
@@ -21,20 +21,18 @@
     "edgedb": "^1.3.2"
   },
   "devDependencies": {
-    "@types/jest": "^29.5.1",
-    "@types/node": "^20.1.7",
-    "conditional-type-checks": "^1.0.5",
-    "edgedb": "^1.1.0",
-    "esbuild": "^0.15.7",
-    "globby": "^13.1.2",
+    "@types/jest": "^29.5.2",
+    "@types/node": "^20.3.2",
+    "conditional-type-checks": "^1.0.6",
+    "edgedb": "^1.3.2",
```

Comment:
I couldn't actually find an explanation of what the version string for a workspace actually does, but I thought it made sense to bump this to latest 🤷

Reply:
Yeah, I assume that's what we're trying to do since these are dev dependencies.